src/i18n/translations/dev.json

```diff
@@ -469,5 +470,6 @@
   "vegaWallet": "Vega Wallet",
   "rewardsComingSoon": "Rewards is coming soon",
   "withdrawsComingSoon": "Withdraws are coming soon",
-  "associationChoice": "You have $VEGA tokens held by the vesting contract. Would you like to associate these or $VEGA directly from wallet?"
+  "associationChoice": "You have $VEGA tokens held by the vesting contract. Would you like to associate these or $VEGA directly from wallet?",
```

Comment:
Grammatical error? "Would you like to associate these or $VEGA directly from wallet?"